### DISCUSSION TITLE
Did we lose the ability to preview revisions (instead of just reverting) at some point?

### DISCUSSION THREAD

**Discussion by msimpson** [craftcms/cms#14521] [Q&A] [answered]
My client is wondering why the only option for revisions is "Revert content from this revision".

They'd like to view that revision in some way (and I seem to remember there being options previously).

**Comment by thupsi** [comment]
I think the "revert content from this revision" button is only visible _when you are actually viewing a revision_?

**Comment by thupsi** [comment]
Hmm, it seems though that the "modified" indicators are missing from many fields that should have them, so maybe that's what tripped your client?

**Comment by msimpson** [comment]
I mean view as in using the "Preview" or "View" actions when on a revision.

As in:

![image](https://github.com/craftcms/cms/assets/730334/995db3d2-3608-49b0-898b-3e132219b2bd)

versus,

![image](https://github.com/craftcms/cms/assets/730334/d5de3f15-fbf9-4489-a094-a83c23708263)

when inside the Entry editor.

I seem to remember a time when this was possible.

Specifically when one could preview (in the live editor) or view (in a new tab using an `x-craft-preview` token) a revision.

And so does my client.

Is this some recent change we both missed or am I misremembering functionality?

(Updated the title to clarify.)

**Reply by thupsi** [reply]
Ah, gotcha! I'm not sure actually. Someone else will have to shed some light on this, sorry!

**Comment by brandonkelly** [comment, marked as answer]
It looks like revision previewing got lost in the shuffle as we were moving to the new unified element editor in Craft 4.

I’ve just added it back for 4.9, as well as the next Craft 5 Beta release (cdcaf5d8a8f623c4a97ee08cf0598d9cd9e21643). Decided not to bring back Live Preview though, since that is a little awkward for non-editable elements. So it will just be the “View” button.

**Reply by brandonkelly** [reply]
5.0.0-beta.6 is out with that change.

**Reply by brandonkelly** [reply]
Craft 4.9.0 is out now with the same fix.
